autorest/codegen/templates/sample.py.jinja2

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+{{ code_model.options['license_header'] }}
+
+{{ imports }}
+"""
+The sample just shows how to use the method and may not run successfully.
+# PREREQUISITES
+{% if "credential" in client_params %}
```

Comment:
we don't want `azure-identity` if it's an AzureKeyCredential. This should be a check if we're doing a TokenCredential

Reply:
yeah, AzureKeyCredential is in `azure.core`, not `azure.identity`